core/test/uix/core_test.cljs

```diff
@@ -208,5 +208,65 @@
       (is (= "<button title=\"hey\">hey</button>"
              (t/as-string ($ as {:title "hey"} "hey")))))))
 
+(defonce *error-state (atom nil))
+
+(def error-boundary
+  (uix.core/create-error-boundary
+   {:derive-error-state (fn [error]
+                          {:error error})
+    :did-catch          (fn [error info]
+                          (reset! *error-state error))}
+   (fn [state [children]]
```

Comment:
since in UIx2 components are single-arity functions taking a map of props, this part has to be updated as well. Instead of a vector of args the error boundary should pass in a map of props.

Reply:
Okay cool, so then the `render-fn` args would be identical to that of `uix.core/fn` right?

Reply:
@chrisetheridge see the example below https://github.com/pitch-io/uix/pull/98#issuecomment-1446676215